misc/python/materialize/cloudtest/k8s/cockroach.py

```diff
@@ -117,8 +122,12 @@ def generate_stateful_set(self) -> V1StatefulSet:
         )
 
 
-COCKROACH_RESOURCES = [
-    CockroachConfigMap(),
-    CockroachService(),
-    CockroachStatefulSet(),
-]
+def create_cockroach_resources(
```

Comment:
I think all the `create_` methods are misnamed. They do not create the resources, so the `create_` prefix should be removed altogether.

Reply:
True!

✅ Renamed.